File: Programming Basics/JavaScript/Conditional Statements/Labs/area-of-figures.js
function solve(input) {
    const figure = input[0];

    let area = 0;
    if (figure == "square") {
        const a = Number(input[1]);
        area = a * a;
    } else if (figure == "rectangle") {
        const a = Number(input[1]);
        const b = Number(input[2]);
        area = a * b;
    } else if (figure == "circle") {
        const r = Number(input[1]);
        area = Math.PI * r * r;
    } else if (figure == "triangle") {
        const a = Number(input[1]);
        const h = Number(input[2]);
        area = (a * h) / 2;
    }

    console.log(area);
}

File: Programming Basics/C++/Conditional Statements/Labs/AreaOfFigures/AreaOfFigures.cpp
#include <iostream>

#define _USE_MATH_DEFINES
#include <math.h>

using namespace std;

int main()
{
    string figure;
    cin >> figure;

    double area = 0;
    if (figure == "square")
    {
        double a;
        cin >> a;

        area = a * a;
    }
    else if (figure == "rectangle")
    {
        double a, b;
        cin >> a >> b;

        area = a * b;
    }
    else if (figure == "circle")
    {
        double r;
        cin >> r;

        area = M_PI * r * r;
    }
    else if (figure == "triangle")
    {
        double a, h;
        cin >> a >> h;

        area = a * h / 2;
    }

    cout.setf(ios::fixed);
    cout.precision(3);
    cout << area << endl;
}
